apps/chronos/.env.example

```
@@ -31,14 +31,15 @@ CHRONOS_ENTRA_CLIENT_SECRET=FILL_ME_IN
-# CHRONOS_SMTP_PORT=587
-# CHRONOS_SMTP_USER=
-# CHRONOS_SMTP_PASS=
-# CHRONOS_SMTP_FROM_NAME=Filc
-# CHRONOS_SMTP_FROM_EMAIL=noreply@filc.hu
-# CHRONOS_SMTP_SECURE=false
+# SMTP configuration (Mailpit in the local dev compose stack)
+# See compose.yml — web UI http://localhost:8025, SMTP localhost:1025
+CHRONOS_SMTP_HOST=localhost
+CHRONOS_SMTP_PORT=1025
+CHRONOS_SMTP_USER=
+CHRONOS_SMTP_PASS=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CHRONOS_SMTP_PASS key should go before the CHRONOS_SMTP_PORT key

(UnorderedKey)